sdk/eventhub/event-hubs/rollup.base.config.js

```diff
@@ -117,11 +117,15 @@ export function browserConfig(test = false) {
 
       nodeResolve({
         mainFields: ["module", "browser"],
-        preferBuiltins: false
+        preferBuiltins: false,
+        dedupe: ["buffer", "events", "util", "process", "assert"]
       }),
 
       cjs({
-        namedExports: { events: ["EventEmitter"] }
+        namedExports: {
+          events: ["EventEmitter"],
+          assert: ["ok", "deepEqual", "equal", "fail", "deepStrictEqual", "notDeepEqual"]
```

Comment:
Is there any side affect of having these included for the non-test browser rollup?

Reply:
cc @bterlson, @chradek

Reply:
Is it necessary to create named exports for these? Looking through some of the test files I don't see `assert` being imported, unless it's indirectly imported by another dependency. Were you getting `not exported` errors like described in the [README](https://github.com/Azure/azure-sdk-for-js#issues-with-rollup)?

I suspect this only impacts the non-test bundle if the non-test code imports `assert` somewhere, but would need to look at the generated bundle to be sure.

Reply:
Yes, I had received errors such as deepEqual cannot be found on undefined, and was why I had tried adding this and it had gotten me somewhat closer to resolving it.
https://github.com/Azure/azure-sdk-for-js/blob/2de21f340b82d259ddbc01497402c3664a0b81aa/sdk/eventhub/event-hubs/test/misc.spec.ts#L6 is where assert gets imported.

Looking back, the assert import was missing from devDependencies and looks like de-duping it in node-resolve plugin is the only fix needed. Updating to remove these.